bindings/python/examples/full_udu_test.py

```diff
@@ -357,9 +357,32 @@
 
     time.sleep(1 / 60)
 
+#######################################################################
+# now lets add some an addon :P
+
+# tell the manager about current device setup
+device_list = MANAGER_UDU_MSG_t.pack(
+    20,  # HobovrManagerMsgType::Emsg_uduString
+    7,   # 6 devices - 1 hmd, 2 controllers, 3 trackers
+    0, 6969,  # device description
+    1, 6969,  # device description
+    1, 6969,  # device description
+    2, 6969,  # device description
+    2, 6969,  # device description
+    2, 6969,  # device description
+    3, 69,  # device description
```

Comment:
`    3, 69,  # device description`
Missed a 69 for the port?

Reply:
Honestly i don't remember, haven't touched python code in ages...